R/utils.R

```diff
@@ -55,63 +55,6 @@ NULL
 #     return(x)
 # }
 
-#' Convert various input as pattern to a vector used in tokens_select,
-#' tokens_compound and kwic.
-#' @inheritParams pattern
-#' @inheritParams valuetype
-#' @param concatenator concatenator that join multi-word expression in tokens object
-#' @param levels only used when pattern is a dictionary
-#' @param remove_unigram ignore single-word patterns if `TRUE`
-#' @seealso [pattern2id()]
-#' @keywords internal
-pattern2list <- function(x, types, valuetype, case_insensitive,
```

Comment:
Can you create a temporary internal function that maps to the old one? We need this to work for the first version at least:
https://github.com/quanteda/quanteda.textstats/blob/4a7ee5fdf5534b814585379125ceadf87b79c20c/tests/testthat/test-utils.R#L10

Reply:
Added in 1ab1816